Add entity_status to Campaign, fix pep8 issues.

josephtyler opened the pull request:
- Add `entity_status` to Campaign object
- Update campaign fixtures
- Fix pep8 issues

Fixes #132 and #134.

juanshishido commented on tests/fixtures/campaigns_all.json:
```diff
@@ -17,6 +17,7 @@
       "total_budget_amount_local_micro": null,
       "id": "2wap7",
       "paused": false,
```
Let's remove all instances of `paused`

juanshishido commented on twitter_ads/campaign.py:
```diff
@@ -138,6 +143,7 @@ class Campaign(Resource, Persistence, Analytics, Batch):
 resource_property(Campaign, 'created_at', readonly=True, transform=TRANSFORM.TIME)
 resource_property(Campaign, 'updated_at', readonly=True, transform=TRANSFORM.TIME)
 resource_property(Campaign, 'deleted', readonly=True, transform=TRANSFORM.BOOL)
+resource_property(Campaign, 'entity_status', readonly=True)
```
`entity_status` is writable on both campaigns and line items.

josephtyler commented:
Ok, I've added `entity_status` to line_items as well, added some line_item tests, and updated the fixtures for both line_items and campaigns to remove the `paused` field.

Looks like someone with write access needs to merge. Thanks!
